.github/workflows/conda.yml

```diff
@@ -0,0 +1,17 @@
+name: conda CI
+
+on:
+  pull_request:
+    # run only if environment.yml changed
+    paths:
+      - 'environment.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker://continuumio/miniconda3:4.8.2
```

Comment:
```suggestion
      - uses: docker://nfcore/base:latest
```

Reply:
`nfcore/base` is essentially the same container, and is the one we'll be using for the main build. So may as well use it here as well I figure..